OWML.Patcher/GameVersionReader.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+
+namespace OWML.Patcher
+{
+    public class GameVersionReader
+    {
+        private readonly BinaryPatcher _binaryPatcher;
+
+        private const int PlayerSettingsSector = 0;
+        private const string VersionPattern = @"\d+(\.\d+)+";//can handle 1-infinite dots
```

Comment:
How many results does this give? There are dates tn the file that would give wrong results, no?

Reply:
Not in that sector

Reply:
For reference this sector is 0x1000-0x1257 (LeftStick_X_PC)